server/routerlicious/packages/lambdas/src/nexus/index.ts

```diff
@@ -298,6 +300,25 @@ export function configureWebSocketServices(
 				[BaseTelemetryProperties.correlationId]: correlationId,
 			};
 
+			if (enableNetworkCheck) {
```

Comment:
Maybe even the flag can be made more explicit. Something like enablePrivateLinkNetworkCheck.